build.gradle

```diff
@@ -27,11 +36,15 @@ sourceCompatibility = 1.8
 allprojects {
     dependencies {
         // IPJ
-        compile "com.github.hyperledger.iroha-java:client:master-SNAPSHOT"
+        compile('com.github.hyperledger.iroha-java:client:master-SNAPSHOT')
+        // Spring
+        compile('org.springframework.boot:spring-boot-starter-web')
```

Comment:
I think we can exclude few unwanted dependencies like `hibernate`. We can also use `jetty` instead of build in `tomcat`.